internal/cache/gache/gache_test.go

```diff
@@ -397,69 +379,62 @@ func Test_cache_Delete(t *testing.T) {
 		fields     fields
 		want       want
 		checkFunc  func(want) error
-		beforeFunc func(args)
+		beforeFunc func(args, *cache)
 		afterFunc  func(args)
 	}
 	defaultCheckFunc := func(w want) error {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           key: "",
-		       },
-		       fields: fields {
-		           gache: nil,
-		           expireDur: nil,
-		           expireCheckDur: nil,
-		           expiredHook: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           key: "",
-		           },
-		           fields: fields {
-		           gache: nil,
-		           expireDur: nil,
-		           expireCheckDur: nil,
-		           expiredHook: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "Call Delete when gache is empty",
+			args: args{
+				key: "vdaas",
+			},
+			fields: fields{
+				gache:          gache.New(),
+				expireDur:      1 * time.Second,
+				expireCheckDur: 1 * time.Second,
+				expiredHook:    nil,
+			},
+			checkFunc: defaultCheckFunc,
```

Comment:
Could you please check if the object was dleted in to the `fields.gache`?

Reply:
@hlts2  sorry, I could not understand what you mentioned 😢 
Could you teach me more detail?

Reply:
ok!
For example, if we call `Delete` method, `Delete` method of `c.gache.Delete` is called.
So we need to check the objects in the gash are gone.

Reply:
is it the responsibility of `gache` library? (same as `file/watcher` we discussed before)